MakingItBetterG55/src/application/EventDisplayer.java

```diff
@@ -0,0 +1,235 @@
+package application;
+
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.BorderPane;
+import javafx.scene.text.Font;
+import javafx.scene.text.FontWeight;
+import javafx.stage.Modality;
+import javafx.stage.Stage;
+import javafx.stage.StageStyle;
+
+public class EventDisplayer {
+	private String title;
+	private String desc;
+	private String[] options;
+
+	// Feel free to change as much as this as you want, the primary goal was to serve as
+	// a graphical foundation for the event popup system.
+	public EventDisplayer(String title, String description, String[] options) {
+		this.title = title;
+		this.desc = splitDesc(description);
+		this.options = options;
+		openPopup(title, desc, options);
+	}
+
+	// Splitting up the description with a new line every 7 words

```

Comment:
Can't be every 7 words because then 7 short words in a row or 7 long ones could be much too short or long. Either set the max width to a reasonable value and setWrapText true, or count the chars themselves up to a certain number then when the number is reached go backwards until you reach the first space and split the line there

Reply:
That makes way more sense, thanks! Done.